ansible/roles/ufw/tasks/proxy_ufw.yml

```diff
@@ -0,0 +1,35 @@
+# Rules for monoliths
+- name: allow web traffic
+  ufw:
+    rule: allow
+    direction: in
+    port: "{{ item }}"
+    proto: tcp
+  with_items:
+    - 443  # https
+    - 80   # http
+  when: is_monolith
+
+- name: allow ssh traffic
+  ufw:
+    rule: allow
+    direction: in
+    port: 22
+    proto: any
+  when: is_monolith
+
+- name: allow mosh traffic
+  ufw:
+    rule: allow
+    direction: in
+    port: "60000:61000"
```

Comment:
This is a big port range to have open to the public. See comment above about public vs VPN interfaces. That seems ideal. (However, I also agree that this is better than what we had before.)